initfiles/etc/DIR_NAME/environment.conf.in

```diff
@@ -21,7 +21,9 @@ interface=*
 use_epoll=true
 # allow kernel pagecache flushing where enabled (true/false)
 allow_pgcache_flush=true
-
```

Comment:
This is so new configs will get these settings.  Configmgr creates the Environment section for new configs from what is in the conf file.  It is in the xml.in so the initial/default xml has these settings, because the initial xml file is not generated from configmgr.